src/i18n.js

```diff
@@ -221,6 +221,115 @@ const resources = {
       couldNotDelete: "Não foi possível apagar",
     },
   },
+  fr: {
+    translation: {
+      medicalReport: "Rapport médical",
+      bulletinCreated: "Bulletin créé",
+      numberAttendanceCode: "Code de présence doit être un nombre",
+      numberPatientCode: "Code patient doit être un nombre",
+      nameSize: "Taille du nom doit comporter au moins 2 caractères",
+      consciousnessLevelBlank: "Niveau de conscience ne peut pas être vide",
+      diuresisBlank: "Diurèse ne peut pas être vide",
+      feverBlank: "fièvre ne peut pas être vide",
+      generalConditionBlank: "Condition générale ne peut pas être vide",
+      doctorNameBlank: "Nom du médecin ne peut pas être vide",
+      bloodPressureBlank: "Pression artérielle ne peut pas être vide",
+      respirationBlank: "Respiration ne peut pas être vide",
+      birthdayBlank: "Anniversaire ne peut pas être vide",
+      obsSize: "L'observation doit comporter au moins 6 caractères",
+      accessDenied: "Accès refusé",
+      fillPatCodeAndAttendance: "Veuillez remplir 'Code patient' et 'Présence'",
+      patientCode: "Code patient",
+      attendanceCode: "Code de présence",
+      birthday: "Date d'anniversaire",
+      patientName: "Nom du patient",
+      generalCondition: "Conditions générales",
+      bloodPressure: "Pression artérielle",
+      consciousnessLevel: "Niveau de conscience",
+      fever: "Fièvre",
+      respiration: "Respiration",
+      diuresis: "Diurèse",
+      observations: "Observations",
+      doctorName: "Nom du médecin",
+      signedAt: "Signé à",
+      add: "Ajouter",
+      normal: "Normal",
+      serious: "Sérieux",
+      stable: "Écurie",
```

Comment:
"stable" here I guess means constant so the right traduction in french is "stable". An "écurie" is where horses are living.